lib/std/private/osfiles.nim

```diff
@@ -396,6 +396,9 @@ proc moveFile*(source, dest: string) {.rtl, extern: "nos$1",
   ##
   ## Can be used to `rename files`:idx:.
   ##
+  ## In Linux, if `dest` already exists, it will be atomically swapped,
+  ## therefore it is possible to "prepare" an empty file to speed up I/O performance for moving huge files.
```

Comment:
Is this *guaranteed* across distributions, C library runtimes, and filesystem types?

Reply:
> If newpath already exists, it will be atomically replaced

See link, it is for Linux glibc. Feel free to suggest better wording.